cluster-autoscaler/core/scale_down_test.go

```diff
@@ -1080,6 +1080,58 @@ var defaultScaleDownOptions = config.AutoscalingOptions{
 	MaxMemoryTotal:                   config.DefaultMaxClusterMemory * units.GiB,
 }
 
+func TestDaemonSetEvictionForEmptyNodes(t *testing.T) {
+	options := config.AutoscalingOptions{
```

Comment:
WDYT about making this test table-driven and adding a case e.g. for when creating evictions times out?